perun-wui-core/src/main/java/cz/metacentrum/perun/wui/model/columnProviders/VoColumnProvider.java

```diff
@@ -59,11 +60,20 @@ public boolean canSelectObject(Vo object) {
 	public PerunDataGrid.PerunFilterEvent<Vo> getDefaultFilterEvent() {
 		return new PerunDataGrid.PerunFilterEvent<Vo>() {
 			@Override
-			public boolean filterOnObject(String text, Vo object) {
+			public boolean filterOnObject(Set<PerunColumnType> columnTypeSet, String text, Vo object) {
 				if (object != null) {
-					if (object.getName().toLowerCase().startsWith(text.toLowerCase()) ||
-							object.getShortName().toLowerCase().startsWith(text.toLowerCase()))
+					if (columnTypeSet.isEmpty() && object.getName().toLowerCase().contains(text.toLowerCase())) {
```

Comment:
You should be more defensive and also check if ``columnTypeSet`` is not null, otherwise your IF will throw NullPointerException. Also previously default filtering was done on both "name" and "shortName" params, but now, when no column is passed, only name is compared for filtering. This default should correspond with pre-checked options on dropdown itself in each view/page. Please update this in all ColumnProviders where you modified default filter.